addons/account_edi_facturx/models/account_edi_format.py

```diff
@@ -324,13 +324,14 @@ def _find_value(xpath, element=tree):
 
                         # Product.
                         name = _find_value('.//ram:SpecifiedTradeProduct/ram:Name', element)
-                        if name:
-                            invoice_line_form.name = name
                         invoice_line_form.product_id = self_ctx._retrieve_product(
                             default_code=_find_value('.//ram:SpecifiedTradeProduct/ram:SellerAssignedID', element),
                             name=_find_value('.//ram:SpecifiedTradeProduct/ram:Name', element),
                             barcode=_find_value('.//ram:SpecifiedTradeProduct/ram:GlobalID', element)
                         )
+                        # force original line description instead of the one copied from product's Sales Description
+                        if name:
+                            invoice_line_form.name = name
```

Comment:
Note that the import is overriden in account_edi_ubl_cii, so this should be changed in both places